spyder/widgets/tests/test_editor.py

```diff
@@ -106,11 +107,35 @@ def editor_folding_bot(base_editor_bot):
 @pytest.fixture
 def editor_splitter_bot(qtbot):
     """Create editor splitter."""
-    editor_splitter = EditorSplitter(None, Mock(), [], first=True)
-    qtbot.addWidget(editor_splitter)
-    editor_splitter.show()
-    yield editor_splitter
-    editor_splitter.destroy()
+    es = editor_splitter = EditorSplitter(None, Mock(), [], first=True)
+    qtbot.addWidget(es)
+    es.show()
+    yield es
+    es.destroy()
+
+
+@pytest.fixture
+def editor_splitter_layout_bot(editor_splitter_bot):
+    """Create editor splitter for testing layouts."""
+    es = editor_splitter_bot
+
+    # Allow the split() to duplicate editor stacks.
+    def clone(editorstack):
+        editorstack.close_action.setEnabled(False)
+        editorstack.set_introspector(Mock())
+        editorstack.set_find_widget(Mock())
+        editorstack.set_io_actions(Mock(), Mock(), Mock(), Mock())
+        editorstack.new('foo.py', 'utf-8', 'a = 1\nprint(a)\n\nx = 2')
+        editorstack.new('spam.py', 'utf-8', 'print(spam)')
```

Comment:
Please change `spam` for another word, I'm not quite sure but maybe this is related with the find in files test failing.

Reply:
Done.